include/openrave/kinbody.h

```diff
@@ -330,6 +332,39 @@ class OPENRAVE_API KinBody : public InterfaceBase
         bool _bVisible = true; ///< if true, geometry is visible as part of the 3d model (default is true)
         bool _bModifiable = true; ///< if true, object geometry can be dynamically modified (default is true)
 
+        struct CalibrationBoardParameters { ///< used by GT_CalibrationBoard
+            int numDotsX; ///< number of dots in x direction, minimum 3
+            int numDotsY; ///< number of dots in y direction, minimum 3
+            dReal dotsDistanceX; ///< distance between center of dots in x direction
+            dReal dotsDistanceY; ///< distance between center of dots in y direction
+            RaveVector<dReal> dotColor; ///< color of dot mesh, which can differ from the color of the board mesh
+            std::string patternName; ///< string that determines pattern of big and normal dots
+                                     /// currently, the only supported type is "threeBigDotsDotGrid"
+                                     /// any unsupported pattern will generate a full grid of normal dots
+            dReal dotDiameterDistanceRatio; ///< dot diameter divided by minimum of dot distances x and y
+            dReal bigDotDiameterDistanceRatio; ///< big dot diameter divided by minimum of dot distances x and y
+            ///< trimesh representation of the collision data of calibration board dot grid in this local coordinate system
+            /// Shares the same caveats as _meshcollision.
+            TriMesh _dotmeshcollision;
```

Comment:
- `TriMesh _dotmeshcollision;`  -> not sure why necessary unless somehow vision algorithms use this data...? if it is just for rendering, then you don't need it.